app/javascript/dashboard/mixins/time.js

```diff
@@ -12,5 +12,29 @@ export default {
       const unixTime = fromUnixTime(time);
       return formatDistanceToNow(unixTime, { addSuffix: true });
     },
+    shortTimestamp(time) {
+      const timestamp = time;
```

Comment:
Any reason to proxy this variable? The replace method returns a new string anyway